src/elements/image/ImageElementForm.tsx

```diff
@@ -94,3 +97,26 @@ const ImageView = ({ fieldViewSpec }: ImageViewProps) => {
     </div>
   );
 };
+
+type CustomDropdownViewProps = {
+  fieldViewSpec: CustomFieldViewSpec<string, Array<Option<unknown>>>;
```

Comment:
Could we replace this `unknown` type w/ e.g. `string` (I think that's the right type here?)

Reply:
I was in two minds about this - I wanted it to be clear that the value here can be anything, e.g. a number or boolean (we later `JSON.stringify` it before using it as a property in the DOM).

In this instance though, they're all strings - so your suggestion would make sense.

Reply:
Oh, that's interesting! We could always widen that type to a union of `string|number|boolean` to accommodate those types – or even better, do some type magic to infer this. V. happy to pair on this if you'd like – but happy to leave as a string for now, also.

Reply:
Resolved w/ a string type for now – as with conversation on #54, we think it'd be better to defer these sorts of things to the consumer.